src/style.css

```diff
@@ -1 +1,20 @@
 /* styles go here */
+
+body {
+  max-width: 600px;
+  padding: 0 px 30px;
+}
+
+.inputWrapper:not(:last-child) {
+  margin-bottom: 10px;
+}
+
+.inputGroup:not(:last-of-type) {
+  margin-bottom: 20px;
+}
+
+/* stylelint-disable-next-line selector-attribute-quotes */
+input[type=submit] {
```

Comment:
it would be better to add a specific class for input like `submit-button` and use here to add styles

Reply:
Hello, I have just corrected the problem, sent it back for checking, passed all the checks in VS (
✔ Found 1 files
✔ Files analyzed

✨  There's no problem, good job �
null

--- Mate: html linter ---

✨✨ Your HTML is fine! ✨✨
Linting CSS files
StyleLint Passed

Linting HTML files
✔ Found 1 files
✔ Files analyzed

✨  There's no problem, good job 👏
HTMLLint Passed ) and GitHub says: All checks have failed/

Reply:
Hi, according to [instruction point 21](https://mate-academy.github.io/layout_task-guideline/#how-to-solve-the-layout-tasks-on-github) after corrections you should re-request review by mentor
Only after that HW will go to list `on review`
<img width="333" alt="Screenshot 2022-05-04 at 14 29 37" src="https://user-images.githubusercontent.com/94289038/220103886-b645df99-6a79-4229-a1df-88d9e0dc0ee9.png">

Reply:
also do not forget to provide demo links in PR descriptions